testdata/BUILD

```diff
@@ -677,6 +677,24 @@ py3_image(
     main = "py3_image.py",
 )
 
+container_image(
+    name = "py3_image_base_with_custom_run_flags",
+    base = "@py3_image_base//image",
+    cmd = ["-V"],
```

Comment:
why do you set command here?

Reply:
Because for py3 images, for example, the entry point is python, if I don't supply something after it, it will hang.